lib/data-access/bookmarks.ts

```
@@ -55,87 +61,89 @@ async function fetchExternalBookmarks(): Promise<UnifiedBookmark[] | null> {
   return inFlightBookmarkPromise;
 }
 
+async function fetchExternalBookmarksWithRetry(retries = 3, delay = 1000): Promise<UnifiedBookmark[] | null> {
+  for (let attempt = 0; attempt < retries; attempt++) {
+    if (attempt > 0) {
+      console.log(`[DataAccess/Bookmarks] Retry attempt ${attempt}/${retries - 1} for fetching bookmarks`);
+    }
+    const result = await fetchExternalBookmarks();
+    if (result && result.length > 0) return result;
+    await new Promise(res => setTimeout(res, delay * Math.pow(2, attempt)));
```

Biome (1.9.4)

[error] Use the '**' operator instead of 'Math.pow'.

Unsafe fix: Use the '**' operator instead of 'Math.pow'.


(lint/style/useExponentiationOperator)